README.md

```diff
@@ -1 +1,183 @@
 # cypress-selector-generator
+
+Tired of the boilerplate needed for writing unambiguous selectors for Cypress UI tests? Use `cypress-selector-generator` to create navigation shorthands!
```

Comment:
This line & the first line have the wrong package name

Reply:
For artistic flow - I think you also need another beat.
```suggestion
Are ambiguous Cypress UI test selectors driving you batty? Does the thought of writing boilerplate needed for writing *un*ambiguous selectors for your tests leave you exhausted?  Then get equipped with `cypress-selector-shorthand` and start creating navigation shorthands with absolute ease!
```

Only thing better would be a GIF montage of SDETs flailing on a keyboard with informercial levels of competence.

Reply:
Presumably you're being facetious?

Reply:
I mean, only if you think you can't make the GIF montage happen.
Consider it an artistic flow suggestion more than anything else. :smile: